src/java/fr/paris/lutece/util/signrequest/IPAuthentificator.java

```diff
@@ -0,0 +1,72 @@
+package fr.paris.lutece.util.signrequest;
+
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+
+public class IPAuthentificator implements RequestAuthenticator {
+	
+	private List<String> _listIPs;
```

Comment:
Maybe use a Set ?

Reply:
A set will be more adequate, changed in the last commit